boot.php

```diff
@@ -844,7 +853,7 @@ function check_addons(App $a)
  */
 function killme()
 {
-	exit();
+	Process::stop(Process::STOP_EXIT);
```

Comment:
Please don't make use of `killme()`, this is a deprecated function that will be removed in a future version.

Reply:
Puh .. Than I've to check & replace **all** of the `killme()` everywhere in Friendica..
I thought maybe we can make it with another PR

Reply:
I've seen you replaced some `killme()` with `Process::stop()` but then you reverted the change. Is there a specific reason?

Reply:
Fair enough.

Reply:
Then just bring up another PR with replacement of deprecated function calls with new.